packages/ra-keycloak/src/authProvider.ts

```diff
@@ -120,9 +120,12 @@ export const keycloakAuthProvider = (
             }
         }
         await initKeyCloakClient(keycloakClient, options.initOptions);
-        return keycloakClient.logout({
+        keycloakClient.logout({
             redirectUri,
         });
+        // The Keycloak client will (forcefully) do its own redirection, so we need to disable
+        // React-admin's redirection
+        return false;
```

Comment:
This is just cleanup to make it clear that the Keycloak js client is responsible for the redirection, not RA